Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Generated Random Locators

## Changelog:
- Generated random locators so army, building, siege models are visible and placed on the map now.


# How to test:
Open the game, make sure armies, sieges, buildings have models placed on the map.
